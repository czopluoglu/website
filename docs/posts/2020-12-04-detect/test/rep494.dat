00101110111011111110
01101101100111111111
10011010011011001101
01011101011100100111
11001000101111101111
00000000010000000001
01110011011110010000
11010011101100101110
11000000101000100011
10010101111001111101
10000000011010101011
00010011101010101110
00001000101000110011
10000100001010100100
00000000001110111010
01110011111110001111
00000001101101101010
11101111101001001101
01110101110101011111
11010001110110101111
00100000011010010000
10000000011101101111
01001001001111001011
00000001101011000111
10101011011011110001
10001001100010100000
10100001001110101111
00011011010000101010
10101001011011111110
01100111000100110111
10101011011010001001
01000100110010010110
10100101111011100111
11110010111111111011
00011001110000011100
10111100111001010010
10001110101011111110
00010001011100101110
00101000011111111111
00101001101011010001
00000101110011101001
00000000111111111110
00100010111000100000
00000000001100001111
10001001011110001010
11111101111111101100
11101000010010100010
11111111111111000111
11011100110000001110
10010000000011110000
01111110111111111111
00001000001011110110
11011010100001010111
10010011111111101111
10101011111111101111
10001011100000101100
01100100100100011111
11001101011110111111
01111001010010111101
01110101011010111111
00001100010010000100
10001101010111100010
11010111011001010010
10000101010010100111
11011100010010010101
10001001000010001011
10110101101010000110
10101000011110111111
11101011110110111011
00010000000001010101
00110001111001100100
11101101101011111011
11011111111111101111
10100101111000000000
10110010101111011111
01111111011000101010
01000000001101011110
11101011111011011111
11101110111111101001
11111100111111111011
01110010001010001100
01000001001011111110
10010001001001000100
11111101011111111111
11111101110110110011
11110010111000001000
11011000001011101100
10100101001000111101
11101100100101000010
10001010001110111101
10001001110100000000
01000001001100001000
11100101100000110010
00110001110010111010
11101011101100101101
10100100011110011110
00011101110101101001
01000101011100000100
01010101111111000111
11010011100011101000
10011110000111001010
00100000000000001001
10110111100010011111
10110001100010000011
00000001010111100111
11110101010111001111
11010101001110101111
00100001111111101110
10001001101111110111
01011001111011101100
10110111111001100111
00110111111010111111
01011111011101111011
00000000001101100111
00000001001000011110
11110110111110010111
10111101111111010110
10000001100010000110
00111011111010000000
11000110011111101110
00111110111011110011
11011001011000111111
00100110100001000000
11111011000110100100
00001101000111101111
01001000000100111010
11101101111101110110
01101000100000100101
00001101010100000001
00010001111100000110
11010101111000000010
11011111010000101100
11011110010000000000
00000000001110110011
00001101001110111110
00000001001010100010
10001001011110010110
11001000011111111111
00001000101110111111
00010011000000010111
00110111111110111001
00010000101010011010
11000001001110100101
01001001101000110100
01111011111100110111
00001000110001010110
11011101110111101111
11000010100110100110
11111111111000100010
11110010101010001110
00010001011100001110
10111011101010111111
10000000011101010111
01011011111111111110
11000100000011000110
01010011011101000000
00001000001110000110
11001101000011000001
11111110101010111110
01001111010000011011
11000001011110101110
10001000101001110011
11001101111110001000
11010100000110100110
11101101001100101110
00000001011000010010
11110111011111110010
00000001110110001001
11100101101111111010
10110101001011100110
00111101111100100111
01111111001111111111
11001100101111111111
11010111101000011111
00000001011110000000
10001011110001010001
10000001001101000111
11111001111111000011
00011011111011100010
00111101101000010111
01001100001010001011
10111101111000000110
00010100111010111111
01101001000010001010
10100110101101000111
11011101111111101101
10001000010000000011
10001001101011000111
10000111111111111111
10010000101010111111
00000000001110100011
01001001111010111111
10011000001010101110
10111001111111101110
10110001111111101111
00001000011100111111
10101011011010110111
01110001011000101101
00100011101111101011
10011010000010000011
10000111111111101111
10110000101101100110
11011111001111011111
10000001101000001100
00100010000011110110
10100111111111111111
11101101111110111011
11011101011101110110
01011100110101111011
10000000111110100101
00011000111100101110
11111111111110110111
11011010111110100110
10011000000111101110
10011111111111111101
10001101101111011111
00101001111001101111
10100101011000100111
00101011000110001010
00100101110010111111
00000001100000100110
10101001010010101000
11001000101100100011
10000111111111110110
00000000010110100011
11110110110010000001
00110011111010011111
00000001100000101010
01001110011101110110
00001000101100100110
01110111111010111111
10011000000001100100
00001101011010111111
00100100001101000011
00000001011111111010
01000000001000000110
11011111111010010000
01011110001010111110
11011111111111110000
01110111111111001100
11011000111010111100
10000110001001110010
01000111001011111010
10001000111001000110
00101100011010011110
01000000010010000010
10100101100111111110
10101101111110101110
11111001111000101111
00010011111100111110
