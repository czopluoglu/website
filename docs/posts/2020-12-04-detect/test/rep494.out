-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep494.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.25
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  3
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.9669
                   IDN index value:   0.7947
                           Ratio r:   0.7153

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    1    2    2    1    1    2    2    2    2
    3    3    3    3    3    3    3    3    3    3

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    2    5    6

   -----------CLUSTER  2-------------

    3    4    7    8    9   10

   -----------CLUSTER  3-------------

   11   12   13   14   15   16   17   18   19   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+++-++-+---++------
   +d++-++----+----+---
   ++d------+---+++----
   ++-d++++-+---+-+----
   ---+d+++++--+------+
   ++-++d++++-------+--
   ++-+++d-+------+----
   ---+++-d++------+--+
   +---++++d+---+------
   --++++-++d++--------
   ---------+d+-++++++-
   -+-------++d+++-++++
   +---+------+d++++-++
   +-++----+-+++d++--++
   --+-------++++d++++-
   --++--+---+-+++d++++
   -+-----+--+++-++d+++
   -----+----++--+++d++
   ----------++++++++d+
   ----+--+---+++-++++d
 
 No cross validation for this DETECT run
